profiles/portal/etc/supervisor/conf.d/xmlrpc-daemon.conf

```diff
@@ -1,5 +1,5 @@
 [program:xmlrpc]
```

Comment:
Maybe we can take this opportunity to rename this services to kamrpc instead of xmlrpc (Filename and this header)